src/main/java/com/mathworks/ci/MatlabBuilder.java

```diff
@@ -441,9 +546,7 @@ private synchronized int execMatlabCommand(Run<?, ?> build, FilePath workspace,
             String matlabFunctionName =
                     FilenameUtils.removeExtension(Message.getValue(MATLAB_RUNNER_TARGET_FILE));
```

Comment:
I'm still familiarizing myself with the code here - is the matlabFunctionName either "batch" or "run" depending on the MATLAB version? Or is it something else?

Reply:
I believe yes. We use "-batch" command if the MATLAB release is 18b or above and "-r" before that.